### DISCUSSION TITLE
LILYGO® TTGO T-Camera recording volume low and delay

### DISCUSSION THREAD

**Discussion by AST-esp-rasp** [pschatzmann/arduino-audio-tools#105] [Q&A] [answered]
Hi I have the streams-i2s-webserver_wav running on a LILYGO® TTGO T-Camera.
It occures to me that the volume is low this probably can be solved using an better amplifyer.
But can I do somthing on the input side? 
the sound has a delay of approx. 2 or  3 seconds can this time be reduced?

**Comment by pschatzmann** [comment, marked as answer]
- For the delay please have a look at https://github.com/pschatzmann/arduino-audio-tools/wiki/Performance. Decreasing bufferes is dicreasing delays at the risk that the signal might be breaking off if it is too small
- For the volume: The AudioServer has a setConverter() method. You could try to use a ConverterScaler with a factor  > 1 and a max_value of 32767. See https://github.com/pschatzmann/arduino-audio-tools/wiki/Volume-Control
